nixos/modules/services/security/aesmd.nix

```diff
@@ -25,6 +25,22 @@ in
       default = false;
       description = lib.mdDoc "Whether to build the PSW package in debug mode.";
     };
+    environment = mkOption {
+      type = with types; attrsOf str;
+      default = { };
+      description = mdDoc "Additional environment variables to pass to the AESM service.";
+      # Example environment variable for `sgx-azure-dcap-client` provider library
+      example = {
+        AZDCAP_COLLATERAL_VERSION = "v2";
+        AZDCAP_DEBUG_LOG_LEVEL = "INFO";
+      };
+    };
```

Comment:
I don't think we need that and should advice people to use the option directly instead.

Reply:
You think they should use `config.systemd.services.aesmd.environment`? I don't think that's very ergonomic. How should we advice people?